File: Unidad 3/Meta 3.1/Python/SistemaBiologico.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from typing import List

class SistemaBiologico(object):
	def __init__(self, nombre: str):
		self._nombre = nombre

	def getNombre(self) -> str:
		return self._nombre

	def setNombre(self, nombre: str):
		self._nombre = nombre

	def __str__(self) -> str:
		return "SistemaBiologico{" + "nombre=" + self._nombre + '}'

File: Unidad 2/Meta 2.1/C#/Ecosistema/SistemaBiologico.cs
using System;


public class SistemaBiologico
{
    private string nombre;

    public SistemaBiologico(string nombre)
    {
        this.nombre = nombre;
    }

    public string Nombre
    {
        get { return this.nombre; }
        set { this.nombre = value; }
    }

    override public string ToString()
    {
        return "SistemaBiologico{" + "nombre=" + this.nombre + '}';
    }

}

